telemetry/telemetry.go

```diff
@@ -26,7 +21,7 @@ var (
 	caCert            = flag.String("ca_crt", "", "CA certificate for client certificate validation. Optional.")
 	serverCert        = flag.String("server_crt", "", "TLS server certificate")
 	serverKey         = flag.String("server_key", "", "TLS server private key")
-	zmqAddress        = flag.String("zmq_address", fmt.Sprintf("tcp://localhost:%d", swsscommon.GetORCH_ZMQ_PORT()), "ZMQ address")
+	zmqAddress        = flag.String("zmq_address", "", "ZMQ address")
```

Comment:
In usage, explain what is the behavior of missing this flag, and of empty string?

Reply:
Fixed.